bin/alluxio-start.sh

```diff
@@ -261,8 +260,7 @@ start_worker() {
   fi
 
   echo "Starting worker @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-  (ALLUXIO_WORKER_JAVA_OPTS=${ALLUXIO_WORKER_JAVA_OPTS} \
-     nohup ${BIN}/launch-process worker > ${ALLUXIO_LOGS_DIR}/worker.out 2>&1 ) &
+  ( nohup ${BIN}/launch-process worker > ${ALLUXIO_LOGS_DIR}/worker.out 2>&1 ) &
```

Comment:
remove spaces after `(` and before `)`

Reply:
There's no consistency on this file about spacing with parentheses. Feel like this should be handled by checkstyle or a separate pr.